rhamt/utils/ftp.py

```diff
@@ -578,14 +578,12 @@ class FTPClientWrapper(FTPClient):
     """
 
     def __init__(self, entity_path=None, entrypoint=None, host=None, login=None, password=None):
-        credentials = conf.get_config("credentials")
         env = conf.get_config("env")
-        ftp_data = env.ftpserver
-        host = host or ftp_data.host
-        login = login or credentials[ftp_data.credentials]["username"]
-        password = password or credentials[ftp_data.credentials]["password"]
+        host = host or env.ftpserver.host
+        login = login or env.ftpserver.credentials.password
+        password = password or env.ftpserver.credentials.username
```

Comment:
The fields got switched : 

     login = login or credentials[ftp_data.credentials]["username"]
    password = password or credentials[ftp_data.credentials]["password"]